ci/build.js

```diff
@@ -28,4 +29,7 @@ ci.sh('mvn -B clean install -DskipTests -s ci/settings.xml');
 ci.stage('UNIT TESTS');
 ci.sh('mvn -B verify -s ci/settings.xml');
 
+// Remove 'gpg' config
+ci.sh('rm -rf /home/circleci/.gnupg');
```

Comment:
If you want to clean it, let's do it properly and put it in a `try {} finally {}`. Otherwise it won't be cleaned if one of the previous commands fail.